cmd/flarectl/dns.go

```diff
@@ -24,10 +24,6 @@ func formatDNSRecord(record cloudflare.DNSRecord) []string {
 }
 
 func dnsCreate(c *cli.Context) {
-	if err := checkEnv(); err != nil {
```

Comment:
I think this cleanup 💯 but do wonder if we could move it to a single `app.Before` instead of each individual flag 🤔

Reply:
I tried it and while looking a lot cleaner, it breaks the usecase of `flarectl ips` which does not require any credentials so i think we are left with sticking it onto the groups of subcommands that does require credentials :)